tests/utils/test_github_project_queries.py

```diff
@@ -71,3 +76,5 @@ def test_get_project_field_options_query():
     actual_query = get_project_field_options_query(organization_name, repository_name, project_number)
 
     assert actual_query == expected_query
+    leftover_placeholders = re.findall(r'\{\w+\}', actual_query)
```

Comment:
Here I tried to add 4th {param} into constant and the test fails on KeyError.
I believe the logic have to be improved to handle it better when some typo or additional param will appear in the input (constant.)

Idea: contact is defined on another location than logic which works with it. It can happen that logic will not be updated. (Yeah, Should not happen.)

Reply:
See the newest version of the code, which solves your suggestions.